Realm.xcodeproj/xcshareddata/xcschemes/RealmSwift.xcscheme

```diff
@@ -64,6 +64,14 @@
                BlueprintName = "RealmSwift Tests"
                ReferencedContainer = "container:Realm.xcodeproj">
             </BuildableReference>
+            <SkippedTests>
+               <Test
+                  Identifier = "ProjectedCollectTestsTemplate">
```

Comment:
Should these actually be skipped here? If these are base classes that shouldn't be run then this won't work for spm.

Reply:
fixed

Reply:
This test case appears to not just not exist at all, so this doesn't need to be here.